### PR TITLE
feat/get talks from db

### PR DESCRIPTION
**Pull request by KenanHaciyev**

### CODE REVIEW

File: projects/core/src/lib/core.module.ts [hunk 1]
```diff
@@ -51,7 +50,7 @@ import { UserStoreFacade } from "./Store/users/users.store.facade";
 		HomePageComponent,
 		InfoPanelComponent
 	],
-	exports: [CoreComponent, HeaderComponent, FooterComponent, InfoPanelComponent],
+	exports: [CoreComponent, HeaderComponent, FooterComponent, InfoPanelComponent, ButtonComponent],
```
Comment (Tsurhanau) [inline projects/core/src/lib/core.module.ts:53]: ButtonComponent should not be exported from CoreModule

File: projects/common-components/src/lib/talks/talks-list/talks-list.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="jsmu-talks-page">
 	<jsmu-talk-item
 		*ngFor="let talk of talks | sort; let i = index"
-		[isShowLine]="isShowLine(i)"
+		[isShowLine]="isShowLine(talks | sort, i)"
```
[the remark this answers is not in the document]
Comment (KenanHaciyev) [reply projects/common-components/src/lib/talks/talks-list/talks-list.component.html]: done

File: projects/core/src/lib/components/talks-page/talks-page/talks-page.component.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { Observable } from "rxjs";
+
+@Component({
+	selector: "jsmu-talks-page",
+	templateUrl: "./talks-page.component.html",
+	styleUrls: ["./talks-page.component.scss"],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TalksPageComponent implements OnInit {
+	public talks!: Observable<Talk[]>;
```
[the remark this answers is not in the document]
Comment (KenanHaciyev) [reply projects/core/src/lib/components/talks-page/talks-page/talks-page.component.ts]: done

File: projects/core/src/lib/core.module.ts (hunk 1 above)
Comment (KenanHaciyev) [reply projects/core/src/lib/core.module.ts]: done

**Review by Tsurhanau — APPROVED** [review]

**Pull request merged**